superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -50,6 +52,44 @@ interface DatabaseModalProps {
 
 const DEFAULT_TAB_KEY = '1';
 
+const StyledDBModal = styled(StyledModal)`
+  .ant-alert {
+    margin-top: ${({ theme }) => theme.gridUnit * 4}px;
+    color: #325D7E;
```

Comment:
I saw that you're using `Alert` `info`. Why are you changing the default `color`, `fontSize` and `border`? I'm worried that we'll have a lack of theme consistency.